overlord/confdbstate/confdbmgr.go

```diff
@@ -159,15 +160,18 @@ func readViewIntoChange(chg *state.Change, tx *Transaction, view *confdb.View, r
 
 	result, err := GetViaView(tx, view, requests)
 	if err != nil {
-		if errors.Is(err, &confdb.NoDataError{}) || errors.Is(err, &confdb.NoMatchError{}) {
-			apiData["confdb-error"] = err.Error()
-			chg.Set("api-data", apiData)
-			return nil
+		if !errors.Is(err, &confdb.NoDataError{}) {
+			return fmt.Errorf("cannot read confdb %s/%s: %w", tx.ConfdbAccount, tx.ConfdbName, err)
```

Comment:
shouldn't this be prefixed with "internal error:" then?

Reply:
Yes, I've added this now